### PR TITLE
Additional mutation only filter

### PR DESCRIPTION
**Pull request by AmmarHusain-TomTom**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,8 @@ target/
 .ipr
 
 .DS_Store
+.classpath
+.project
+.settings/org.eclipse.core.resources.prefs
+.settings/org.eclipse.jdt.core.prefs
+.settings/org.eclipse.m2e.core.prefs
```
Comment (nikhilmohite-tomtom) [inline .gitignore:14]: Not sure if target is by default ignored. if not may be we can add:
`*/target/`

minor:
we can also add below two lines, for IntelliJ users:

`.idea/`
`*.iml`

**Review by nikhilmohite-tomtom — APPROVED** [review]
LGTM!

(one minor comment for .gitignore file)

File: .gitignore (hunk 1 above)
Comment (AmmarHusain-TomTom) [reply .gitignore:14]: `target` & IntelliJ exclusions are already there

**Review by poojamamidwar-tomtom — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (nikhilmohite-tomtom) [reply .gitignore:14]: ohh, my bad.
It was in collapsed section.
Thank you !

**Pull request merged**